pkg/runtest/run_test.go

```diff
@@ -108,6 +115,72 @@ func test(t *testing.T, sysTarget *targets.Target) {
 	}
 }
 
+func runTest(req *RunRequest, executor string) {
+	cfg := new(ipc.Config)
+	sysTarget := targets.Get(req.P.Target.OS, req.P.Target.Arch)
+	cfg.UseShmem = sysTarget.ExecutorUsesShmem
+	cfg.UseForkServer = sysTarget.ExecutorUsesForkServer
+	cfg.Timeouts = sysTarget.Timeouts(1)
+	cfg.Executor = executor
+	var env *ipc.Env
+	defer func() {
+		if env != nil {
+			env.Close()
+		}
+	}()
+	for run := 0; run < req.Repeat; run++ {
+		if run%2 == 0 {
+			// Recreate Env every few iterations, this allows to cover more paths.
```

Comment:
Can we reuse `env.ForceRestart()` here as well?

Reply:
Done.